client/src/components/leftSidebar/topLeftLogoAndTitle.js

```diff
@@ -7,12 +7,13 @@ import Logo from "../framework/logo";
 @connect(state => ({
   responsive: state.responsive,
   datasetTitle: state.config?.displayNames?.dataset ?? "",
+  aboutLink: state.config?.links?.["about-dataset"],
```

Comment:
the name `link` is misleading.  As currently defined, this must be an absolute URL.    Suggest that you rename this to `aboutURL` or something similar.